CONTRIBUTING.md

```diff
@@ -15,31 +15,46 @@ tox                      # runs 'format', 'lint', and 'unit' environments
 
 This charm is used to deploy Superset Server in a k8s cluster. For local deployment, follow the following steps:
 
-## Set up your development environment
+## Set up your development environment with Multipass [~ 10 mins]
+When you’re trying things out, it’s good to be in an isolated environment, so you don’t have to worry too much about cleanup. It’s also nice if you don’t need to bother too much with setup. In the Juju world you can get both by spinning up an Ubuntu virtual machine (VM) with Multipass, specifically, using their Juju-ready `charm-dev` blueprint.
+```
+sudo snap install multipass
+multipass launch --cpus 4 --memory 8G --disk 30G --name superset-vm charm-dev
+```
+That's it! Your dependencies are all set up, please run `multipass shell superset-vm` and skip to `Create a model`.
+
+## Set up your development environment without Multipass blueprint
 ### Install Microk8s
 ```
-# Install Microk8s from snap:
-sudo snap install microk8s --classic --channel=1.25
+# Install microk8s from snap:
+sudo snap install microk8s --channel=1.27-strict/stable
 
-# Add the 'ubuntu' user to the Microk8s group:
-sudo usermod -a -G microk8s ubuntu
+# Setup an alias for kubectl:
+sudo snap alias microk8s.kubectl kubectl
 
-# Give the 'ubuntu' user permissions to read the ~/.kube directory:
-sudo chown -f -R ubuntu ~/.kube
+# Add your user to the Microk8s group:
+sudo usermod -a -G snap_microk8s $USER
 
-# Create the 'microk8s' group:
-newgrp microk8s
+# Switch to the 'microk8s' group:
+newgrp snap_microk8s
+
+# Wait for microk8s to be ready:
+microk8s status --wait-ready
 
 # Enable the necessary Microk8s addons:
-microk8s enable hostpath-storage dns
+sudo microk8s.enable dns rbac hostpath-storage
```

Comment:
You get a deprecation warning about this now. You are supposed to run enable one-by-one.

Reply:
Thanks, updated to do this one at a time